model/task/test_result.go

```diff
@@ -74,6 +74,12 @@ func getMergedTaskTestResults(ctx context.Context, env evergreen.Environment, ta
 	var baseTasks []Task
 	if output.TestResults.Version == 0 && getOpts != nil {
 		baseTasks = getOpts.BaseTasks
+		if len(baseTasks) > 0 {
```

Comment:
nit: can we leave a comment about why we're checking for this (because of the edge case described in pr)